pages/governance.md

```diff
@@ -0,0 +1,179 @@
+
+### Developer chatroom
+
+The Steering Council will maintain a developer chatroom for developer-focused
+discussions about JuMP.
+
+If you are hacking on JuMP and have a question about the internals, this is the
+best place to go. If you are a user needing general help, we ask that you use
+the [Community forum] instead.
+
+The current chatroom is the [JuMP-dev Gitter channel](https://gitter.im/JuliaOpt/JuMP-dev).
+
+### Monthly developer call
+
+The Steering Council will maintain a monthly developer call to discuss JuMP-related
+business.
+
+The calls are currently scheduled on the fourth Thursday of every month at 14:00
+Eastern. For information on how to take part, join the [Developer chatroom] and
+ask `@odow` or `@mlubin` for an invite to the monthly developer call.
+
+### JuMP-dev workshops
+
+Given suitable demand, the Steering Council will choose a Chair to organize a
+JuMP-dev workshop, at which community members will be invited to present their
+latest work utilizing JuMP.
+
+Previous workshops include [Santiago, 2019](https://jump.dev/meetings/santiago2019/),
+[Bordeaux, 2018](https://jump.dev/meetings/bordeaux2018/), and [Boston, 2017](https://jump.dev/meetings/mit2017/).
+
+Announcements of future workshops will be communicated through the
+[Community forum] and via the [jump.dev](https://jump.dev) website.
+
+## Decision making process
+
+### Technical
+
+Technical decisions, such when a pull request is ready to be accepted and
+merged, are made by consensus via the relevant Github issues and pull requests.
+If consensus cannot be achieved, the community member proposing the change may
+be invited by the Steering Council to present their proposal at a Monthly
+developer call for further discussion and community input.
+
+Due to his role in the creation of JuMP, Miles Lubin holds the role of [BDFL](https://en.wikipedia.org/wiki/Benevolent_dictator_for_life).
+If consensus cannot be achieved, the final decision shall be made by the BDFL.
+
+### Other
+
+All other decisions to be made, e.g., the date and location for a JuMP-dev
+workshop, should be tabled as an agenda item and discussed on a Monthly
+developer call. If consensus cannot be achieved, the final decision shall be
+made by the Steering Council.
+
+## Commit rights
+
+Commit rights are managed through [Github permissions](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization)
+in the [JuMP-dev Github organization].
+
+There are two levels of commit rights: repository maintainers and core
+contributors.
+
+For security, all maintainers and core contributors must have two-factor
+authentication enabled through their Github account.
+
+### Repository Maintainer
+
+Repository maintainers have `Maintain` rights to an individual repository as an
+outside collaborator. This allows them to push code to a branch in the
```

Comment:
This is a good reason for standardization. I took a look and the outside collaborators have a mix of `Write`, `Maintain`, and `Admin` rights.

Reply:
Here's the breakdown: https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#repository-access-for-each-permission-level

The short answer is yes.

The main difference between Maintain and Admin is the ability to edit settings (archive, default branches, change deploy keys), invite external collaborators, delete issues, etc.

I think Maintain is the one we want. We don't want external collaborators adding new collaborators, or deleting the repository, for example. 

For people transferring repositories, there is a trade-off: transfer to JuMP-dev and get structure, help with maintenance, etc, and loose some permissions such as the ability to add collaborators.